external/fv3fit/fv3fit/_shared/pytorch/Graphloss.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import dataclasses
+
+
+def _standard_mse():
+    loss = torch.nn.MSELoss()
+
+    def custom_loss(y_true, y_pred):
+        return loss(y_pred, y_true)
+
+    return custom_loss
+
+
+def _standard_mae():
+    loss = torch.nn.L1Loss()
+
+    def custom_loss(y_true, y_pred):
+        return loss(y_true, y_pred)
+
+    return custom_loss
```

Comment:
As the code is here, calling `_standard_mae()` is identical to just calling `torch.nn.L1Loss()`. Is that intentional? If it is, can you delete these functions and just call e.g. `L1Loss` directly below?